installation.html

```diff
@@ -11,312 +11,216 @@
-        select the volumes you want to mount <br/>
-        choose the 'Guest' radio button and connect <br/>
-      </code>
-    </p>
-  </li>
-  <li>
-    <p>
-      Once mounted, will appear as /Volumes/jhipster. Create a symlink to make the volume accessible in the local "~/jhipster" folder.
-    </p>
-    <p>
-      <code>
-        ln -s /Volumes/jhipster ~/jhipster
-      </code>
-    </p>
-  </li>
-</ol>
-</p>
-<h3>SSH configuration</h3>
-<p>
-You can now connect to your docker container with SSH. You can connect as "root/jhipster" or as "jhipster/jhipster", and we recommand you use the "jhipster" user as some of the tool used are not meant to be run by the root user.
-</p>
-<p>
-Start by adding your SSH public key to the Docker container:
+</ul>
 </p>
 <p>
-<code>
-cat ~/.ssh/id_rsa.pub | ssh -p 4022 jhipster@localhost 'mkdir ~/.ssh && cat >> ~/.ssh/authorized_keys'
-</code>
-</p>
+    <code>
+        docker run --name jhipster -v ~/jhipster:/home/jhipster/app:rw -v ~/.m2:/home/jhipster/.m2:rw -p 8080:8080 -p
+        3000:3000 -p 3001:3001 -d -t jdubois/jhipster-docker
+    </code>
+    <b>PLEASE NOTE : </b> If you have already started the container once before, you do not need to run above command,
+    you can simply start/stop existing container.
 <p>
-You can now connect to the Docker container:
+<ul>
+    <li>To stop the container execute :
+        <code>
+            docker stop jhipster
+        </code>
+    </li>
+    <li>And to start again, execute :
+        <code>
+            docker start jhipster
+        </code>
+    </li>
+</ul>
 </p>
-<p>
+Also in case you update the docker image (rebuild or pull from dockerhub), it's better to remove existing container, and run the container all over again.
+To do so, first stop the container and then remove it and run again. i.e. :
+<br/>
 <code>
-ssh -p 4022 jhipster@localhost
+    <ol>
+        <li>docker stop jhipster</li>
+        <li>docker rm jhipster</li>
+        <li>docker pull jdubois/jhipster-docker</li>
+        <li>docker run --name jhipster -v ~/jhipster:/home/jhipster/app:rw -v ~/.m2:/home/jhipster/.m2:rw -p 8080:8080
+            -p 3000:3000 -p 3001:3001 -d -t jdubois/jhipster-docker
+        </li>
+    </ol>
 </code>
-</p>
-<h3>Your first project</h3>
 <p>
-You can then go to the /jhipster directory in your container, and start building your app inside Docker:
-</p>
 <p>
-<code>
-cd /jhipster<br/>
-yo jhipster
-</code>
+    <br/><br/>
+    <b>Please note : </b> On linux, you might need to run the docker command as sudo user if your user is not part of
+    docker group. It's a good idea to add your user to docker group so that you can run docker commands as non-root
+    user. Follow steps on <a href="http://askubuntu.com/a/477554">http://askubuntu.com/a/477554</a> to do so.
 </p>
+
+<h3>Accessing the container</h3>
 <p>
-  If the following exception occurs, you need to change the owner of the /jhipster directory. See next step.
```

Comment:
I made some tests. This error can occur when the local directory isn't created !
So we should keep this !